packages/labs/motion/package.json

```diff
@@ -31,19 +31,119 @@
     "/position.{d.ts,d.ts.map,js,js.map}"
   ],
   "scripts": {
-    "start": "web-dev-server -- --node-resolve --watch --preserve-sym-links --open demo/index.html",
-    "build": "npm run clean && npm run build:ts --build && rollup -c",
-    "build:watch": "rollup -c --watch",
-    "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
-    "build:ts:watch": "tsc  --build --watch",
-    "clean": "rm -rf {index,animate,position,animate-controller}.{js,js.map,d.ts} development/ test/ *.tsbuildinfo",
-    "dev": "scripts/dev.sh",
-    "test": "npm run test:dev && npm run test:prod",
-    "test:dev": "MODE=dev node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
-    "test:prod": "MODE=prod node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
-    "test:watch": "npm run test:dev -- --watch",
-    "checksize": "rollup -c --environment=CHECKSIZE",
-    "regen-package-lock": "rm -rf node_modules package-lock.json; npm install"
+    "start": "wireit",
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:ts:types": "wireit",
+    "build:rollup": "wireit",
+    "test": "wireit",
+    "test:dev": "wireit",
+    "test:prod": "wireit",
+    "checksize": "wireit"
+  },
+  "wireit": {
+    "start": {
+      "command": "web-dev-server -- --node-resolve --watch --preserve-sym-links --open demo/index.html",
+      "dependencies": [
+        "build"
```

Comment:
For a TODO: Can this be made to just depend on :ts? Should be faster to iterate on. The demos already import local files from `development/`, we'd just need export conditions in a WDS config.

Reply:
Added comment